docs/release-notes/0.7/release-0.7.md

```diff
@@ -0,0 +1,52 @@
+# .NET for Apache Spark 0.7 Release Notes
+
+### New Features and Improvements
+
+* Expose `SparkContext.setLogLevel()` to change log level programmatically ([#360](https://github.com/dotnet/spark/pull/360))
+* Expose `Schema()` for `DataFrameReader` and `DataStreamReader`([#248](https://github.com/dotnet/spark/pull/248))
+* Expose `Column.Apply()` ([#323](https://github.com/dotnet/spark/pull/323))
+* "Deprecated" annotation is added to the following functions:
+  * `WindowSpec.RangeBetween(Column start, Column end)`
+  * `Window.RangeBetween(Column start, Column end)`
+  * `Functions.UnboundedPreceding()`
+  * `Functions.UnboundedFollowing()`
+  * `Functions.CurrentRow()`
+
+### Breaking Changes
+* None
```

Comment:
Should the removal of *ActiveSession() APIs be mentioned also ?

Reply:
Yes! let me update it.